### PR TITLE
Update ingestion-pipeline.yaml to add the ingestion glue job

### PR DESCRIPTION
**Pull request by ElhadiCherifi95**

### CODE REVIEW

File: infra/ingestion-pipeline.yaml [hunk 1]
```diff
@@ -72,6 +131,24 @@ Resources:
...
+    Type: "AWS::Glue::Job"
+    Properties:
+      Role: !Ref AWSGlueJobRole
+      Name: "GlueDataIngestionJob"
+      Command: {
+        "Name" : "glueetl",
+        "ScriptLocation": !Sub "s3://${ArtifactBucketName}/${ETLScriptsPrefix}/process_data_ingestion.scala"
+      }
+      DefaultArguments: {
+          "--database_name" : !Ref DatabaseName,
```
Comment (AlbanPhelip) [inline infra/ingestion-pipeline.yaml]: Mettre "--job-language": "scala"

File: infra/ingestion-pipeline.yaml [hunk 2]
```diff
@@ -16,16 +16,75 @@ Parameters:
...
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "s3:GetObject"
+                  - "s3:PutObject"
+                  - "s3:ListBucket"
+                  - "s3:DeleteObject"
+                Resource:
+                  - !Sub "arn:aws:s3:::${IncomingBucketName}"
```
Comment (AlbanPhelip) [inline infra/ingestion-pipeline.yaml]: RawBucketName
PreparedBucketName
SchemaBucketName

Comment (ElhadiCherifi95) [reply infra/ingestion-pipeline.yaml]: okey

File: infra/ingestion-pipeline.yaml (hunk 1 above)
Comment (ElhadiCherifi95) [reply infra/ingestion-pipeline.yaml]: +1

**Pull request merged**